### ISSUE TITLE
Localize main website keyword, meta title and meta description

### ISSUE THREAD

**Issue by mariannk** [nopSolutions/nopCommerce#43]
We should allow a store owner to localize main website keyword, meta title and meta description by store and by language on the "general and misc" page > seo settings tab (http://admin-demo.nopcommerce.com/Admin/Setting/GeneralCommon). Like it's already done for some entities when have multiple languages enabled. It's also already supported for some payment methods (description).

The problem is that these settings can be also configured per store. Both of these additions (multi-store and localizations) make implementation a bit complex. And how should displayed it in UI? Configure per store and localization at the same time? Any ideas?

Also note that it requires some performance optimization if used for this work item - "Setting" entity should not be loaded (by ID) when loading a localized value - see "GetLocalizedSetting" method of \Nop.Services\Localization\LocalizationExtensions.cs

There's no really good solution in terms of UI/UX

**Comment by cnEUL** [comment]
In my opinion very important. We have many multilanguage pages and when i look at google, it is a mixed language. Title English, Some description german and so on.  Nearby the Homepage has SEO fields with no effects (from Topic). Also other SEO basics are missed.

**Comment by rudgr** [comment]
really need this feature for multi language SEO... any update on this? :)

**Comment by kava81** [comment]
I would like to suggest localization of product image "alt" and "title".
thanks

**Comment by DmitriyKulagin** [comment]
Depends on #5096

**Comment by bo-bac** [comment]
what if we use Topics (Content management - Topics) at least for Home page.
Topics already localized and have SEO meta things.
We need to add 'multi-store'bility here. agree it will add complexity but it could be...

![image](https://user-images.githubusercontent.com/4517566/108630855-b0b86b00-746f-11eb-8a5a-d79750210e51.png)

and for topic item

![image](https://user-images.githubusercontent.com/4517566/108630935-202e5a80-7470-11eb-93d9-86c94ee35916.png)

**Comment by skoshelev** [comment]
Closed #43
